### PR TITLE
feat: seed, tasks model changes, makefile

### PR DESCRIPTION
**Pull request by Tassil0**
- seed
- makefile for easier commands
- tasks tags, priority and asignee are now optional and nullable
- tasks status is now `Pending`, `In progress`, `Done`, default is Pending

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,21 +18,24 @@ TaskVerse is a school project aimed at familiarizing with the Django framework a
...
+
 ### TODO
 
 - [x] tailwindcss
 - [ ] lucide
 - [x] preline UI
 - [ ] preline UI js
-- [ ] ruff
\ No newline at end of file
+- [x] ruff
```
Comment (Katussska) [inline README.md:41]: whats that?

File: tasks/migrations/0003_alter_task_assigned_to.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='task',
+            name='assigned_to',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='assigned_tasks', to=settings.AUTH_USER_MODEL),
+        ),
+    ]
```
Comment (Katussska) [inline tasks/migrations/0003_alter_task_assigned_to.py:21]: co dela ta trida?

File: README.md (hunk 1 above)
Comment (Tassil0) [reply README.md:41]: nainstalovat extension - https://docs.astral.sh/ruff/

File: tasks/migrations/0003_alter_task_assigned_to.py (hunk 2 above)
Comment (Tassil0) [reply tasks/migrations/0003_alter_task_assigned_to.py:21]: migrace co dela zmeny viz PR desc, idk vygenerovalo to django

Comment (Katussska) [reply tasks/migrations/0003_alter_task_assigned_to.py:21]: "viz PR desc" asi jsem dement, ale nevidela jsem to tam. tak jsem byla zmatena

File: README.md (hunk 1 above)
Comment (Katussska) [reply README.md:41]: ou, okey, why not

**Pull request merged**